standards/storing-source-code.md

```diff
@@ -49,3 +42,25 @@ By default, any [ministryofjustice organisation](https://github.com/ministryofju
 
 [template repository]: https://github.com/ministryofjustice/template-repository
 [github actions]: https://github.com/ministryofjustice/github-actions
+
+### GitHub user accounts
+
+If you already have a GitHub account from before you joined MOJ, you can choose whether to continue using it at MOJ or create a new one. Some people prefer to have continuity with previous work; others value the separation.
+
+You must [add your MOJ email address to your
+account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)
+It doesn't matter if it's the 'primary email' or not and it's ok to make it private.
+Maybe [use that email for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)).
+
+Bear in mind that you may wish to also have a home email address added to the
```

Comment:
This is all useful context, but buries the lede a bit. It might be worth opening with something like “If you leave the MOJ, it is your responsibility to either maintain or close your MOJ-specific GitHub account after you leave. We will remove your access to our GitHub organisation, but have no power to close or manager users' accounts.” Then follow that with the bulk of what you've said?

Reply:
I think that means that github will no longer associate commits with that email address to your account, which may not be what everyone wants.

Reply:
@SteveMarshall 

> buries the lede ...
> “If you leave the MOJ, it is your responsibility to either maintain or close your MOJ-specific GitHub account after you leave.

I've changed it to:

> In preparation for when you leave MoJ, it's suggested you add a home email
address to your account as the 'backup email'. The reason is that the account is
yours, not MoJ's, so it remains your responsibility.

I've structured it: "When in $situation, do $thing, because $principle. Specifically $reasons[]". People skim read, so I like to front-load the "hook", and I just think the principle in this case is a bit abstract and not compelling enough, compared to the $situation and $thing we're asking them to do. Maybe I'm overthinking this...

Reply:
> [add] something about removing your @digital.justice.gov.uk address

This seems like a useful item, but since @jennyd raises a concern I suggest a follow-up PR and separate discussion is in order.